File: app/Http/Controllers/SpecialistPatternsJava/Creational/AbstractFactory/AbstractFactoryJava/src/walls/BrickWall.java
package walls;

// Класс BrickWall представляет стену из кирпича.
public class BrickWall implements Wall {

    // Метод build() выполняет построение стены из кирпича.
    @Override
    public void build() {
        System.out.println("Постройка стены из кирпича");
    }
}

File: app/Http/Controllers/SpecialistPatterns/AbstractFactory/Walls/BrickWall.php
<?php

namespace App\Http\Controllers\SpecialistPatterns\AbstractFactory\Walls;

// Класс BrickWall представляет стену из кирпича.
use Illuminate\Support\Facades\Log;

class BrickWall implements Wall {

    // Метод build() выполняет построение стены из кирпича.
    public function build() {
        Log::channel('sergey')->info("Постройка стены из кирпича");
    }
}
